src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -55,3 +55,4 @@ sticker.stationary.body=Теперь вы можете нажать сквозь
 wallpaper.install.title=Набор тематических обоев!
 wallpaper.install.body=ВАЖНО! Обои останутся после удаление этого плагина. Вы можете изменить/убрать фоновой рисунок с помощью действия "Set Background Image".
 wallpaper.install.show-settings=Показать Настройки
+settings.general.content.discreet-mode=Discreet Mode
```

Comment:
@Dragon-0609 I added another one of these. When you get time, could you be a pal and update this, please? [Here is some docs to help you understand the feature better](https://github.com/doki-theme/doki-theme-jetbrains/tree/descreetMode#discreet-mode), use [this link](https://github.com/doki-theme/doki-theme-jetbrains#discreet-mode) if I already merged the pull request.

Feel free to suggest the edit to the pull request if it is still open. Thanks!

Reply:
I have done.